### ISSUE TITLE
Linux launcher always asking to build the game first

### ISSUE THREAD

**Issue by bladehorton** [MorsGames/sm64plus-launcher#4]
If I use the launcher outside or inside of the directory of the built game, it just asks to build the game, and wont let me launch into the settings

**Comment by Anonymous941** [comment]
I'm having the exact some issue on Ubuntu.

**Comment by Razen1994** [comment]
same issue here. Any news on what to do to fix this? I'm on steam deck

**Comment by matthewmiddlebrook** [comment]
I figured out how to get the launcher working on the Steam Deck. It requires a bit of a confusing file structure. The two empty folders can be named whatever.

```
📁
├── 📁 lib (for launcher)
└── 📁
    ├── SM64Plus (launcher executable)
    ├── 📁 assets (for launcher)
    └── 📁 sm64plus (with completed install/build)
```

**Comment by odiplol** [comment]
same issue AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by UntriexTv** [comment]
Just compile the game in bin folder of launcher and it will work

**Comment by LiterallyEarth** [comment]
You have to clone the sm64plus in the bin folder. I also had this problem until I figured it out so I made a pull request for an install script which makes things much easier.

**Comment by yllekz** [comment]
Same issue. The instructions in the launcher are rather unclear and it would be better if it had instructions telling you where to put the files after building the game.

I found that after building sm64plus, you have to move the entire sm64plus folder into the "bin" folder of the sm64plus_x64 repo. This really needs to be in the instructions.

**Comment by FFFluoride** [comment]
Yep, same Issue here, maybe I'll put a PR for the README or something
